<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trang Chính</string>
  </property>
  <property name="styleSheet">
   <string>background-color: #f0f0f0;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblTitle">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>181</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>font-size: 18px; font-weight: bold;</string>
    </property>
    <property name="text">
     <string>Hệ Thống Quản Lý</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnThietBi">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background-color: #4CAF50; color: white; border-radius: 5px; padding: 5px;</string>
    </property>
    <property name="text">
     <string>Quản Lý Thiết Bị</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnKhachHang">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background-color: #2196F3; color: white; border-radius: 5px; padding: 5px;</string>
    </property>
    <property name="text">
     <string>Quản Lý Khách Hàng</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHoaDon">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background-color: #FF9800; color: white; border-radius: 5px; padding: 5px;</string>
    </property>
    <property name="text">
     <string>Quản Lý Hóa Đơn</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnThongKe">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background-color: #9C27B0; color: white; border-radius: 5px; padding: 5px;</string>
    </property>
    <property name="text">
     <string>Thống Kê</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNhanVien">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background-color: #9C27B0; color: white; border-radius: 5px; padding: 5px;</string>
    </property>
    <property name="text">
     <string>Quản Lý Nhân Viên</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDangXuat">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>330</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background-color: #f44336; color: white; border-radius: 5px; padding: 5px;</string>
    </property>
    <property name="text">
     <string>Đăng Xuất</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
